docs/content/docs/getting-started/index.mdx

```diff
@@ -13,45 +13,32 @@ Getting started with BlockNote is quick and easy. Install the required packages
 To install BlockNote with [NPM](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm), run:
 
 ```console tab="npm"
+npm install @mantine/core @mantine/hooks @mantine/utils
 npm install @blocknote/core @blocknote/react @blocknote/mantine
 ```
 
 ```console tab="pnpm"
+pnpm add @mantine/core @mantine/hooks @mantine/utils
 pnpm add @blocknote/core @blocknote/react @blocknote/mantine
 ```
 
 ```console tab="bun"
+bun add @mantine/core @mantine/hooks @mantine/utils
 bun add @blocknote/core @blocknote/react @blocknote/mantine
 ```
 
-BlockNote provides multiple UI library options. Choose the one that best fits your project:
+This will install the [Mantine UI](https://mantine.dev/) version of BlockNote, which works great out of the box and is recommended for new projects.
```

Comment:
I'd prefer to keep the old mantine page and only change the old text from:

> BlockNote provides multiple UI library options. Choose the one that best fits your project:

to:

> The examples in this guide use Mantine as UI library. BlockNote provides multiple UI library options. Choose the one that best fits your project:

the rest of the changes (besides the package install instructions ofc) can then be undone I think.

Reasoning:
- keeps this page super simple (we don't need the mantine styles explanation here
- Gives the sidebar a consistent structure (1 page per UI lib), instead of deviating for the default (mantine) one